src/neuroconv/utils/json_schema.py

```diff
@@ -70,104 +69,93 @@ def get_base_schema(
     return base_schema
 
 
-def get_schema_from_method_signature(method: Callable, exclude: list = None) -> dict:
+def get_schema_from_method_signature(method: Callable, exclude: list[str] | None = None) -> dict:
```

Comment:
Much in the way Pydantic decided to deprecate their generic `.schema` method in favor of explicit/verbose pattern, I think we should too since we deal with many other schemas nearby to this utility and elsewhere in our ecosystem (HDMF, NWB, LinkML, etc. etc.)

Reply:
The deprecation here in favor of `..._json_schema_...` to be explicit about what type of schema we are inferring

Reply:
Ah, got you, yeah I agree it makes sense!